src/js/tabs/osd.js

```diff
@@ -3265,6 +3265,25 @@ osd.initialize = function (callback) {
                     insertOrdered($displayFields, $field);
                 }
 
+                // Add the search field and it's functionality
+                $("#element-fields").prepend(
+                    $(
+                        `<em class="fas fa-search" style="margin-left:3px"></em>
+                        <input type="text" class="osd-elements-search-field" placeholder="${i18n.getMessage("search")}" />`,
+                    ),
+                );
+                $(".osd-elements-search-field").on("input", function () {
```

Comment:
Think OSD is duplicate here
```suggestion
                $(".elements-search-field").on("input", function () {
```

Reply:
Changed